docs/README.md

```diff
@@ -14,6 +14,7 @@ p4c
 │   ├── graphs                -- backend that can draw graphiz graphs of P4 programs
 │   └── bmv2                  -- behavioral model version 2 (switch simulator) back-end
 ├── control-plane             -- control plane API
+|   └──google/rpc             -- Defines error model in `google.rpc` package.
```

Comment:
This is a bit too specific. I would say something like "Definitions of Google Protobuf APIs".

Reply:
Many devices use Protobuf for communication between the hardware device and the control plane and some of the definitions needed are contained in this folder.